app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,76 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if Helpers.is_logged_in?(session)
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/logout' do
+   session.clear
+   redirect to '/login'
+ end
+
+ patch '/tweets/:id' do
+     @tweet = Tweet.find(params[:id])
+   if !params["content"].empty?
+     @tweet.update(:content =>params["content"])
+    #  @tweet.save
+     erb :'tweets/show_tweet'
+   else
+     redirect to "/tweets/#{@tweet.id}/edit"
+   end
+ end
+
+ get '/tweets/new' do
+   if Helpers.is_logged_in?(session)
+     erb :'tweets/create_tweet'
+   else
+     redirect to '/login'
+  end
+ end
+
+ get '/tweets/:id/edit' do
+    if Helpers.is_logged_in?(session)
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?